remote/gitlab/testdata/projects.go

```diff
@@ -160,14 +160,8 @@ var project6Paylod = []byte(`
 	},
 	"archived": false,
 	"permissions": {
-		"project_access": {
-			"access_level": 10,
-			"notification_level": 3
-		},
-		"group_access": {
-			"access_level": 50,
-			"notification_level": 3
-		}
```

Comment:
This payload was not used anywhere. I've just updated it to match this use case.